src/shapefile/render.rs

```diff
@@ -89,8 +90,14 @@ pub fn render(
     let marsh = (0, 10, 220);
     let olive = (194, 176, 33);
 
+    let mut shapetmpfolder = tmpfolder;
+    let batch_process_tmpfolder = PathBuf::from(format!("temp_shapefiles"));
+    if batch {
+        shapetmpfolder = &batch_process_tmpfolder
+    }
+
```

Comment:
a bit more rusty like this:

```suggestion
    let shapetmpfolder = if batch {
        PathBuf::from("temp_shapefiles")
    } else {
       tmpfolder
    };

```